examples/flutter_gallery/lib/demo/transformations/transformations_demo.dart

```diff
@@ -25,17 +25,34 @@ class _TransformationsDemoState extends State<TransformationsDemo> {
     hexagonRadius: _kHexagonRadius,
     hexagonMargin: _kHexagonMargin,
   );
+  bool _seenInstructionDialog = false;
 
   @override
   Widget build (BuildContext context) {
+    if (!_seenInstructionDialog) {
+      Future<void>.delayed(Duration(seconds: 1), () {
```

Comment:
```suggestion
      Future<void>.delayed(const Duration(seconds: 1), () {
```
Makes analyzer happy. 😊

Reply:
On it.  Thanks for the review!